src/main/java/mat/client/util/FeatureFlagConstant.java

```diff
@@ -6,4 +6,5 @@ public interface FeatureFlagConstant {
     String FHIR_BONNIE = "FhirBonnie";
     String MADIE = "MADIE";
     String MADIE_QDM = "MADIE_QDM";
+    String MADIE_QDM_RE_TRANSFER = "MADIE_QDM_RE_TRANSFER";
```

Comment:
In the ticket it is mentioned that to use the feature flag name as: enableQdmRepeatTransfer, shouldn't this be the name for the feature flag